src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.SearchResult.cs

```diff
@@ -90,6 +85,19 @@ private class SearchResult : INavigateToSearchResult
                     parts.Free();
                 }
             }
+
+            public string SecondarySort
```

Comment:
in the non-oop case it def helps.  That's what i'm currently measuring.  For OOP, you're right that it likely won't be noticeably better.